src/go/k8s/cmd/configurator/main.go

```diff
@@ -43,16 +43,32 @@ type configuratorConfig struct {
 	redpandaRPCPort   int
 }
 
+func (c *configuratorConfig) String() string {
+	return fmt.Sprintf("The configuration:\n"+
+		"hostName: %s\n"+
+		"svcFQDN: %s\n"+
+		"configSourceDir: %s\n"+
+		"configDestination: %s\n"+
+		"redpandaRPCPort: %d\n",
+		c.hostName,
+		c.svcFQDN,
+		c.configSourceDir,
+		c.configDestination,
+		c.redpandaRPCPort)
+}
+
 var errorMissingEnvironmentVariable = errors.New("missing environment variable")
 
 func main() {
-	log.Printf("The redpanda configurator is starting")
+	log.Print("The redpanda configurator is starting")
 
 	c, err := checkEnvVars()
 	if err != nil {
 		log.Fatalf("%s", fmt.Errorf("unable to get the environment variables: %w", err))
 	}
 
+	log.Print(c.String())
```

Comment:
if you're implementing Stringer interface you don't need to call string, do you?

Reply:
You are right!. Thank you.